tests/epyccel/test_generic_functions.py

```diff
@@ -350,6 +367,30 @@ def test_mix_int_array():
     f2(x2, a)
     assert np.array_equal( x1, x2 )
 
+@pytest.mark.xfail(sys.platform == 'win32', reason='issue #567 and #568: Duplicated types in a template and header')
+def test_mix_int_array_2():
+    f1 = epyccel(mod2.mix_int_array_2)
+    f2 = mod2.mix_int_array_2
```

Comment:
Are you sure this only fails on windows?
```
@template('g', types=['int[:]', 'int32[:]', 'int64[:]'])
@types('g', 'int')
def mix_int_array_2(x, a):
```
Shouldn't it fail everywhere?

Reply:
yes